cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js

```diff
@@ -401,6 +401,47 @@ function constructSpannerSource(artifactsList, spannerInfo) {
   };
 }
 
+function constructAdlsSource(artifactsList, adlsInfo) {
+  if (!adlsInfo) {
+    return null;
+  }
+
+  let batchArtifact = find(artifactsList, { name: 'adls-plugins' });
+  if (!batchArtifact) {
+    return T.translate(`${PREFIX}.adls`);
```

Comment:
This is not a valid string is it?

Reply:
As seen in your existing code, its seems as a valid string.
[https://github.com/cdapio/cdap/blob/1177ea39c84e0d98a637f68cdec010cdde44ce9e/cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js#L199](url)

Reply:
Corrected string for adls